### ISSUE TITLE
The corresponding path in the train_specularitynet.py was not found under the dataset

### ISSUE THREAD

**Issue by YNKing** [jianweiguo/SpecularityNet-PSD#1]
![image](https://user-images.githubusercontent.com/38598263/143797005-4c1c0602-3192-4277-986d-8fe68bc208a7.png)
In the lines of 31,39,42,46 
/PSD_Dataset/filtered
/PSD_Dataset/aligned
/PSD_Dataset/val
/PSD_Dataset/wild
was not found,Please tell me how to configure the path reasonably

**Comment by aobi12138** [comment]
I have the same problem. Have you already solved it?

**Comment by jianweiguo** [comment]
1. The training data folder in our published dataset (PSD_Dataset/PSD_Train) contains PSD_Train_group, images under this file should be loaded at dataset_filtered (Please see annotation #dataset_aligned: Fixed polarization angles). 
2. The Train folder also contains PSD_Train_diffuse and PSD_Train_specular, where the images correspond to diffuse images (i.e., ground-truth) and specular highlight images. They should be loaded at dataset_filtered (Please see annotation #dataset_ filtered: random polarization angles). 
3. Validation set can be placed in dataset_val.
4. You can test other images outside the dataset in dataset_wild ( you can also do without ground-truth) 

 Of course, you can also rewrite the way of loading the dataset yourself.

**Comment by PhilipLGQ** [comment]
Could you explain more clearly on how we can load the dataset properly? For example, how we rename or relink the directory and how the files under PSD_XX_group should be loaded.

**Comment by cassiopeia-yxx** [comment]
> Could you explain more clearly on how we can load the dataset properly? For example, how we rename or relink the directory and how the files under PSD_XX_group should be loaded.

Please ask you whether solve the problem? If you solve it, please tell me how to rename or relink, Thank you very much!
